migrations/manual/test-educandu-2021-12-13-01-make-cdn-urls-generic.spec.js

```diff
@@ -0,0 +1,48 @@
+import { processSection } from './educandu-2021-12-13-01-make-cdn-urls-generic.js';
+
+const testMarkdown = `
+# Dear World,
+
+here we have a link to [elmu staging](https://staging.cdn.elmu.online/media/my-image.png),
+and this is an image from elmu production:
+![Cat falling from tree](https://cdn.elmu.online/media/falling-cat.jpg)
+
+Here is plain text link to elmu staging (https://staging.cdn.elmu.online/media/smth.pdf), which should not be replaced!
+
+Also a few pictures with URL encoded characters from Open Music Academy:
+
+![Cat falling from tree](https://cdn.integration.openmusic.academy/media/falling%20cat.jpg)
+![Cat falling from tree](https://cdn.staging.openmusic.academy/media/falling%20cat.jpg)
+![Cat falling from tree](https://cdn.openmusic.academy/media/falling%20cat.jpg)
+
+The end!
+`;
+
+const expectedResult = `
+# Dear World,
+
+here we have a link to [elmu staging](cdn://media/my-image.png),
+and this is an image from elmu production:
+![Cat falling from tree](cdn://media/falling-cat.jpg)
+
+Here is plain text link to elmu staging (https://staging.cdn.elmu.online/media/smth.pdf), which should not be replaced!
+
+Also a few pictures with URL encoded characters from Open Music Academy:
+
+![Cat falling from tree](cdn://media/falling%20cat.jpg)
+![Cat falling from tree](cdn://media/falling%20cat.jpg)
+![Cat falling from tree](cdn://media/falling%20cat.jpg)
```

Comment:
Note: This can happen only 4 more times...

Reply:
It took me a while :rofl: